census/tf-single/README.md

```diff
@@ -0,0 +1,121 @@
+# Census: TensorFlow Vanilla Sample
+
+This sample uses the [TensorFlow](https://tensorflow.org) low level APIs and
+[Google Cloud Machine Learning Engine](https://cloud.google.com/ml) to demonstrate
+the single node and distributed TF vanilla version for Census Income Dataset.
+
+## Download the data
+Follow the [Census Income
+Dataset](https://www.tensorflow.org/tutorials/wide/#reading_the_census_data) link to download the data. You can also download directly from [here](https://archive.ics.uci.edu/ml/datasets/Census+Income).
+
+ * Training file is `adult.data`
+ * Evaluation file is `adult.test`
+
+
+## Virtual environment
+There are two options for the virtual environments:
+ * Install [Virtual](https://virtualenv.pypa.io/en/stable/) env
+   * Create virtual environment `virtualenv single-tf`
+   * Activate env `source single-tf/bin/activate`
+ * Install [Miniconda](https://conda.io/miniconda.html)
+   * Create conda environment `conda create --name single-tf python=2.7`
+   * Activate env `source activate single-tf`
+
+
+## Install dependencies
```

Comment:
I would agree but unfortunately navigating the docs for this information can get overwhelming. This is / should be an inexpensive step so spending more than its worth can burden the user.